basyx/aas/adapter/http.py

```diff
@@ -524,6 +550,30 @@ def _namespace_submodel_element_op(cls, namespace: model.UniqueIdShortNamespace,
         except KeyError as e:
             raise NotFound(f"Submodel element with id_short {arg} not found in {namespace!r}") from e
 
+    def _get_submodels(self, request: Request) -> Iterator[model.Submodel]:
+        submodels: Iterator[model.Submodel] = self._get_all_obj_of_type(model.Submodel)
+        id_short = request.args.get("idShort")
+        if id_short is not None:
+            submodels = filter(lambda sm: sm.id_short == id_short, submodels)
+        semantic_id = request.args.get("semanticId")
+        if semantic_id is not None:
+            spec_semantic_id = HTTPApiDecoder.base64urljson(semantic_id, model.Reference, False)
+            submodels = filter(lambda sm: sm.semantic_id == spec_semantic_id, submodels)
+        return submodels
+
+    def _get_submodel(self, url_args: Dict) -> model.Submodel:
+        # TODO: support content, extent parameters
```

Comment:
I think these `# TODO` comments can be removed, as they aren't relevant anymore.